tmt/cli.py

```diff
@@ -203,7 +203,28 @@ def discover(context, **kwargs):
     help='Use specified method for provisioning.')
 @click.option(
     '-i', '--image', metavar='IMAGE',
-    help='Select virtual machine image to be used.')
+    help='Select virtual machine image to use (URI or Box name).')
+@click.option(
+    '-b', '--box', metavar='BOX',
+    help='Vagrant box name to use.')
+@click.option(
+    '-v', '--vagrantfile', metavar='VAGRANTFILE',
+    help='Vagrant box name to use.')
+@click.option(
+    '-m', '--memory', metavar='MEMORY',
+    help='Set memory available to guest in MB.')
+@click.option(
+    '-u', '--user', metavar='USER',
+    help='Username to use for all guest operations.')
+@click.option(
+    '-p', '--password', metavar='PASSWORD',
+    help='Password to use for login into guest system.')
+@click.option(
+    '-k', '--key', metavar='PRIVATE_KEY',
+    help='Private key to use for login into guest system.')
+@click.option(
+    '-r', '--guest', metavar='GUEST',
```

Comment:
if we use guest, let's also use `-g` ?

Reply:
Yes, yes, just an oversight on my part :).

Thanks for noticing.

Reply:
np ;)